lib/paraiso.ex

```diff
@@ -264,6 +264,39 @@ defmodule Paraiso do
       ...>   ]
       ...> )
       {:error, :a, :invalid}
+      iex> Paraiso.process(
```

Comment:
Would you describe this specification in Japanese at near l.243?

Reply:
fixed.